airflow-core/src/airflow/api_fastapi/auth/managers/simple/simple_auth_manager.py

```diff
@@ -283,6 +286,27 @@ def filter_authorized_menu_items(
     ) -> list[MenuItem]:
         return menu_items
 
+    def is_allowed(self, user_id: str, assigned_users: Sequence[HITLUser]) -> bool:
```

Comment:
It seems this one is missing 

```suggestion
    def is_authorized_hitl_task(self, user_id: str, assigned_users: Sequence[HITLUser]) -> bool:
```